src/ProjectTemplates/test/BlazorWasmTemplateTest.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.E2ETesting;
+using Templates.Test.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Test
+{
+    public class BlazorWasmTemplateTest : BlazorTemplateTestBase
+    {
+        public BlazorWasmTemplateTest(ProjectFactoryFixture projectFactory, BrowserFixture browserFixture, ITestOutputHelper output)
+            : base(projectFactory, browserFixture, output)
+        {
+        }
+
+        [Fact]
+        public async Task BlazorWasmStandaloneTemplate_Works()
+        {
+            var project = await ProjectFactory.GetOrCreateProject("blazorstandalone", Output);
+
+            var createResult = await project.RunDotNetNewAsync("blazorwasm");
+            Assert.True(0 == createResult.ExitCode, ErrorMessages.GetFailedProcessMessage("create/restore", project, createResult));
+
+            // We can't run a published standalone app, but let's just make sure it publishes fine
+            var publishResult = await project.RunDotNetPublishAsync();
+            Assert.True(0 == publishResult.ExitCode, ErrorMessages.GetFailedProcessMessage("publish", project, publishResult));
```

Comment:
Could we actually run it by invoking it through the dev-server or similar?

Reply:
It might be worthwhile to refactor the dev-server in to a standalone tool that could be used here. Let me play around with it and I'll file a follow up so we can track updating this once I have something to share.